### PR TITLE
OCPEDGE-1237: IBU Seedgen job updates

### PR DESCRIPTION
**Pull request by jeff-roche**
- Remove build cluster references from the seed images generated on the nightlies
- Add floating tags to the nightlies

### CODE REVIEW

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-release-4.16-nightly-4.16-seedgen-aws-ocp-4-16

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-release-4.16-nightly-4.16-seedgen-aws-ocp-4-16

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-release-4.17-nightly-4.17-seedgen-aws-ocp-4-17 periodic-ci-openshift-kni-lifecycle-agent-release-4.15-nightly-4.15-seedgen-aws-ocp-4-15 periodic-ci-openshift-kni-lifecycle-agent-release-4.16-nightly-4.16-seedgen-aws-ocp-4-16 periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.18-seedgen-aws-ocp-4-18

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-release-4.17-nightly-4.17-seedgen-aws-ocp-4-17 periodic-ci-openshift-kni-lifecycle-agent-release-4.15-nightly-4.15-seedgen-aws-ocp-4-15 periodic-ci-openshift-kni-lifecycle-agent-release-4.16-nightly-4.16-seedgen-aws-ocp-4-16 periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.18-seedgen-aws-ocp-4-18

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-release-4.17-nightly-4.17-seedgen-aws-ocp-4-17 periodic-ci-openshift-kni-lifecycle-agent-release-4.15-nightly-4.15-seedgen-aws-ocp-4-15 periodic-ci-openshift-kni-lifecycle-agent-release-4.16-nightly-4.16-seedgen-aws-ocp-4-16 periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.18-seedgen-aws-ocp-4-18

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-release-4.17-nightly-4.17-seedgen-aws-ocp-4-17

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-release-4.17-nightly-4.17-seedgen-aws-ocp-4-17

**Comment by jeff-roche** [comment]
/pj-rehearse pull-ci-openshift-kni-lifecycle-agent-main-ci-index pull-ci-openshift-kni-lifecycle-agent-main-ci-job pull-ci-openshift-kni-lifecycle-agent-main-images pull-ci-openshift-kni-lifecycle-agent-main-integration pull-ci-openshift-kni-lifecycle-agent-main-security

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.18-seedgen-aws-ocp-4-18 periodic-ci-openshift-kni-lifecycle-agent-release-4.15-nightly-4.15-seedgen-aws-ocp-4-15 periodic-ci-openshift-kni-lifecycle-agent-release-4.16-nightly-4.16-seedgen-aws-ocp-4-16

File: ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-release-4.16__nightly-4.16.yaml [hunk 1]
```diff
@@ -30,9 +30,12 @@ tests:
     dependencies:
       LCA_PULL_REF: pipeline:lca-operator
     env:
+      LCA_PULL_REF_OVERRIDE: quay.io/openshift-kni/lifecycle-agent-operator:4.16
       OCP_BASE_IMAGE_SOURCE: release
       OCP_BASE_VERSION: "4.16"
+      SEED_FLOATING_TAG: "4.16"
       SEED_IMAGE_TAG_FORMAT: nightly
+      SEED_IMAGE_TAG_PREFIX: test
```
[the remark this answers is not in the document]
Comment (jeff-roche) [reply ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-release-4.16__nightly-4.16.yaml:38]: It's no longer needed, I used it to generate a test image for the IBIO team to use. I forgot to remove it

File: ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main__nightly-4.18.yaml [hunk 2]
```diff
@@ -30,8 +30,10 @@ tests:
     dependencies:
       LCA_PULL_REF: pipeline:lca-operator
     env:
+      LCA_PULL_REF_OVERRIDE: quay.io/openshift-kni/lifecycle-agent-operator:4.17
```
[the remark this answers is not in the document]
Comment (jeff-roche) [reply ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main__nightly-4.18.yaml:33]: Yes, there is no 4.18 image of LCA yet

**Comment by eranco74** [comment]
What change removes the build cluster references from the seed images?

**Comment by jeff-roche** [comment]
> What change removes the build cluster references from the seed images?

@eranco74 @giladravid16 it is a combination of the `LCA_PULL_REF_OVERRIDE` which is intended to be used to pull LCA from quay rather than the build cluster and using a released version of OCP

**Pull request merged**